### ISSUE TITLE
Pytorch Geometric training for synthetic data

### ISSUE THREAD

**Issue by dloveland** [RexYing/gnn-model-explainer#29]
Hello,

I am trying to use explain_pyg.py to train the pytorch geometric GCN model specified in models_pyg.py to perform the synthetic task 1. I am using the default args in configs.py but the training seems to get stuck at ~15% accuracy and never gets any better. I did minor hyperparameter tuning, but still no luck. This is significantly different from the train.py and models.py which seem to approach 98% accuracy easily after a few hundred epochs. Is there some issue with the pytorch geometric implementation that I am missing? Has anyone had any luck training pytorch geometric models on the synthetic data? Thanks!

**Comment by dloveland** [comment]
Nevermind, I think the pytorch geometric version does work, there was a bug in how the final test accuracy is calculated.
